.ci/lint_install.sh

```diff
@@ -8,9 +8,13 @@ source "${CI_DIR}/common.sh"
 echo "pwd: $(pwd)"
 ls
 
-# follow deps are already included in ko docker image
-# travis_retry luarocks --local install luacheck
-# travis_retry luarocks --local install lanes # for parallel luacheck
+# follow deps are already included in ko docker image (used in circleci)
```

Comment:
Alternatively, if we want to keep the Travis builds going for now, we could just switch 'em to the Docker image.